src/js/builtins/shell.ts

```diff
@@ -93,6 +154,16 @@ export function createBunShellTemplateFunction(ShellInterpreter) {
       return this.#quiet();
     }
 
+    nothrow(): this {
+      this.#throws = false;
+      return this;
+    }
+
+    throws(doThrow: boolean | undefined): this {
+      this.#throws = typeof doThrow === "boolean" ? doThrow : false;
```

Comment:
```suggestion
      this.#throws = !!doThrow;
```
We could just coerce anything to a boolean.